http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -40,6 +48,7 @@ import scala.jdk.CollectionConverters._
 
 class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
                            helmClient: HelmAlgebra[F],
+                           kubeService: org.broadinstitute.dsde.workbench.google2.KubernetesService[F],
```

Comment:
I think it's right to use `KubernetesService`, but I suspect the implementation is GKE-specific right now. It uses a `com.google.container.v1.Cluster` to create a client: https://github.com/broadinstitute/workbench-libs/blob/develop/google2/src/main/scala/org/broadinstitute/dsde/workbench/google2/KubernetesInterpreter.scala#L434

There should be a way to make this cloud agnostic, will think about this..